### PR TITLE
[bitnami/odoo] remove rollingUpdate from values.yaml

### PR DESCRIPTION
**Pull request by zzorica**
### Description of the change

When setting values of 

```yaml
updateStrategy:
  type: Recreate
```
...deployment fails because in default values.yaml there is:
```yaml
updateStrategy:
  rollingUpdate: {}
 ```
...which gets into template when rendered:
```yaml
updateStrategy:
  type: Recreate
  rollingUpdate: {}
```
..so it fails with:
```
cannot patch "odoo" with kind Deployment: Deployment.apps "odoo" is invalid: spec.strategy.rollingUpdate: Forbidden: may not be specified when strategy `type` is 'Recreate'
```

### Benefits

This will alow to use  `Recreate` updateStrategy so new pods can mount existing PVC not held by old pod anymore.

### Possible drawbacks

None

### Applicable issues

None that I know off.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/odoo/values.yaml [hunk 1]
```diff
@@ -357,7 +357,6 @@ schedulerName: ""
 ##
 terminationGracePeriodSeconds: ""
 ## @param updateStrategy.type Odoo deployment strategy type
-## @param updateStrategy.rollingUpdate Odoo deployment rolling update configuration parameters
```
Comment (juan131) [inline bitnami/odoo/values.yaml:360]:
```suggestion
## @param updateStrategy.rollingUpdate [object,nullable] Odoo deployment rolling update configuration parameters
```

File: bitnami/odoo/values.yaml [hunk 2]
```diff
@@ -369,7 +368,6 @@ terminationGracePeriodSeconds: ""
 ##
 updateStrategy:
   type: RollingUpdate
-  rollingUpdate: {}
```
Comment (juan131) [inline bitnami/odoo/values.yaml:372]:
```suggestion
  rollingUpdate: null
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for this contribution @zzorica !!

I think we should use the "nullable" modifier instead but keep the value and document it. Please check my suggestions.

File: bitnami/odoo/values.yaml (hunk 2 above)
Comment (zzorica) [reply bitnami/odoo/values.yaml:372]: Hey @juan131! TIL! Didn't know it will be ignored if `null`. Nice!

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**